samples/bluetooth/mesh_demo/src/board.h

```diff
@@ -14,7 +14,8 @@ void board_button_1_pressed(void);
 uint16_t board_set_target(void);
 void board_play(const char *str);
 
-#if defined(CONFIG_BOARD_BBC_MICROBIT)
+#if defined(CONFIG_BOARD_BBC_MICROBIT) || \
+	defined(CONFIG_BOARD_BBC_MICROBIT_V2)
```

Comment:
Couldn't `CONFIG_MICROBIT_DISPLAY` be used instead? For all instances of this OR expression